packages/cashier/src/components/cashier-search-box/cashier-search-box.tsx

```diff
@@ -45,14 +60,14 @@ const CashierSearchBox = ({ className, onClear, onSearch, placeholder, search_te
                                     onKeyDown={() => onSearchKeyUpDown(submitForm, search)}
                                     leading_icon={<Icon icon='IcSearch' data_testid='dt_search_icon' />}
                                     trailing_icon={
-                                        search && (
+                                        search ? (
                                             <Icon
                                                 color='general'
                                                 data_testid='dt_close_icon'
                                                 icon='IcCloseCircle'
                                                 onClick={() => onSearchClear(setFieldValue)}
                                             />
-                                        )
+                                        ) : null
```

Comment:
The on-false operand in the ternary operator expression could be `undefined` but `null` is more suitable (source: George). It really seems more logical in a situation where we need rendering nothing to return the code equivalent of nothing (`null`), rather than code equivalent of "I dunno" (`undefined`).